test/test_other.py

```diff
@@ -12298,24 +12301,19 @@ def test_standalone_syscalls(self):
     for engine in config.WASM_ENGINES:
       self.assertContained(expected, self.run_js('test.wasm', engine))
 
-  @parameterized({
-    '': ([],),
-    'assertions': (['-sASSERTIONS'],),
-    'closure': (['-sASSERTIONS', '--closure=1'],),
-    'dylink': (['-sMAIN_MODULE'],),
-  })
-  def test_webgpu_compiletest(self, args):
-    self.run_process([EMXX, test_file('webgpu_jsvalstore.cpp'), '-Wno-error=deprecated', '-sUSE_WEBGPU', '-sASYNCIFY'] + args)
-
   @flaky('https://github.com/emscripten-core/emscripten/issues/25343')
+  @crossplatform  # So tests will run on non-Linux platforms, where FROZEN_CACHE isn't set.
   @also_with_wasm64
   @parameterized({
     '': ([],),
     'closure': (['--closure=1', '-Werror=closure'],),
     'closure_assertions': (['--closure=1', '-Werror=closure', '-sASSERTIONS'],),
   })
   def test_emdawnwebgpu_link_test(self, args):
-    self.run_process([EMXX, test_file('test_emdawnwebgpu_link_test.cpp'), '--use-port=emdawnwebgpu', '-sASYNCIFY'] + args)
+    if config.FROZEN_CACHE:
+      # TODO(crbug.com/446944885): Make Emdawnwebgpu work with FROZEN_CACHE if possible.
+      self.skipTest("test doesn't work with frozen cache")
```

Comment:
Why is this `skipTest` needed now but not before this PR?

Reply:
I think it's because previously the cache was pre-populated with `-sUSE_WEBGPU` but we don't prepopulate it with `--use-port=emdawnwebgpu` (which would have to be done with several different flags too).

I can try to pre-populate the cache with the appropriate Emdawnwebgpu objects if you think that's appropriate.

Reply:
But the old code and the new code here both use `--use-port=emdawnwebgpu`.

Reply:
Oh, sorry, I missed which test this was. I actually have no idea. If I remove emdawnwebgpu from my local cache and run the old test (on `main`), it still fails with `FROZEN_CACHE is set, but cache file is missing`.

Could the `@flaky` be suppressing it?

Reply:
Oh, CI has an explicit separate Embuilder step. Let me figure out how to test that locally and then see where I need to skip tests.

Reply:
... Although I did not figure out exactly why it was passing before and isn't now. `@also_with_wasm64` isn't newly-added.